server/app/assets/javascripts/admin_predicate_configuration.ts

```diff
@@ -250,6 +250,20 @@ class AdminPredicateConfiguration {
       el.closest('label')?.setAttribute('for', newId)
     })
 
+    newRow.querySelectorAll('select').forEach((el: HTMLSelectElement) => {
+      const groupNumString = assertNotNull(el.name.match(/group-(\d+)/))[1]
```

Comment:
Suggestion (non-blocking): there is logic above this for inputs that is identical, consider creating a shared function.

Reply:
Done